api/app.js

```diff
@@ -1,21 +1,26 @@
 const express = require('express');
 const app = express();
 const bodyParser = require('body-parser');
- 
-const mongoose = require('./db/mongoose');
-
+require('dotenv').config();
 
+const port = process.env.PORT || 3000;
 /* ---Load Mongoose Models--*/
-const { List, Task, User } = require('./db/models');
-const jwt = require('jsonwebtoken');
+const {
+    List,
+    Task,
+    User
+} = require('./db/models');
+const {
+    verify
+} = require('./functions/promisifiedFunctions');
 
 /*---Load Middleware---*/
 
 //Response BodyParser Middleware
 app.use(bodyParser.json());
 //Cross-Origin Resource Sharing Middleware
 //https://enable-cors.org/server.html
-app.use(function(req, res, next) {
+app.use((req, res, next) => {
```

Comment:
Made using es6 arrow functions. Assuming you won't need to use the `this` keyword. Just thought it would make it more consistent